Fix crash (abort because of vSmall.avail==0 just after alloc) on 64-bit
machines with 4k page

--- smalltalk-2.1.8/libgst/alloc.c.orig	2003-05-27 19:41:15.000000000 +0200
+++ smalltalk-2.1.8/libgst/alloc.c	2004-07-16 20:00:42.249971627 +0200
@@ -99,7 +99,7 @@
   120, 128, 136, 144, 152, 160, 168, 176, 184, 192, 200,
   208, 224, 232, 248, 264, 288, 312, 336, 368, 400, 448,
   504, 576, 672, 808, 1016, 1352, 2032, 
-  4080 - 2*SIZEOF_CHAR_P, 0
+  4096 - SMALL_OBJ_HEADER_SIZE, 0
 };
 
 static unsigned short sztable[MAX_SMALL_OBJ_SIZE + 1];
